packages/react/src/components/Alert/Alert.tsx

```diff
@@ -59,6 +59,9 @@ export interface AlertProps
   /** Controls Alert's relation to neighboring items. */
   attached?: boolean | 'top' | 'bottom'
 
+  /** An alert can only take up the width of its content. */
+  compact?: boolean
```

Comment:
Why not `fitted`? It feels awkward to have two different terms across the components for the same thing? If we decide that one is better then the other, we can rename all of them at once. What do you think?

Reply:
Good catch, `fitted` will work better there and will be consistent 👍